e2e2/test/cases/neuron/main_test.go

```diff
@@ -55,6 +58,9 @@ func TestMain(m *testing.M) {
 			return ctx, nil
 		},
 		func(ctx context.Context, config *envconf.Config) (context.Context, error) {
+			if !*installDevicePlugin {
+				return ctx, nil
```

Comment:
feels weird to do this, would be nice if we could add the setup func in the check above but ignore if it makes more trouble